python/cudf/cudf/_libxx/table.pyx

```diff
@@ -49,6 +51,10 @@ cdef class Table:
     def _column_names(self):
         return self._data.names
 
+    @property
+    def _index_names(self):
+        return None if self._index is None else self._index._data.names
```

Comment:
Can we just use the `self._index.names` now or is `_index` not guaranteed to be a class of Index / MultiIndex?

Reply:
We could totally use `_index.names`